pkg/features/kube_features.go

```diff
@@ -915,6 +915,13 @@ const (
 	// Speed up container startup by mounting volumes with the correct SELinux label
 	// instead of changing each file on the volumes recursively.
 	SELinuxMount featuregate.Feature = "SELinuxMount"
+
+	// owner: @serathius
+	// beta: v1.30
+	//
+	// Allow watch create a separate watch RPC than watches opened to etcd.
+	// This prevents watch cache from being starved by watches to etcd.
+	SeparateCacheWatchRPC featuregate.Feature = "SeparateCacheWatchRPC"
```

Comment:
no need to re-declare the constant, can use genericfeatures.SeparateCacheWatchRPC below

Reply:
how do we say `Allow watch create a separate watch RPC than watches opened to etcd.` better? (user friendly terms?)

Reply:
Is the entry here for book keeping? looks like what is in `staging/src/k8s.io/apiserver/pkg/features/kube_features.go` is the only one used in the code for checking whether to turn things on or off

Reply:
Done.